domparsing/DOMParser-parseFromString-encoding.html

```diff
@@ -0,0 +1,43 @@
+<!doctype html>
+<title>DOMParser encoding test</title>
+<meta charset="windows-1252"> <!-- intentional to make sure the results are UTF-8 anyway -->
```

Comment:
There should be an assert to ensure this is actually the used encoding for this document.